files/install-deps.yaml

```diff
@@ -13,8 +13,6 @@
           - python3-click
           - git # setuptools-scm
           # httpd & deps
-          - httpd
-          - mod_md
           - mod_ssl
           - nss_wrapper
```

Comment:
I think the `nss_wrapper` is needed only in `setup_env_in_openshift.sh` so can perhaps be removed as well.

Reply:
I wanted to do just that, but then it turned out that `setup_env_in_openshift.sh` is also used in [`run_worker.sh`](https://github.com/packit/packit-service/blob/7011396a299e7addc7328ac7980f8bf21d09c82a/files/run_worker.sh#L9) and [`run_tests.sh`](https://github.com/packit/packit-service/blob/7011396a299e7addc7328ac7980f8bf21d09c82a/files/run_tests.sh#L5). If I'm not mistaken it's required by `fedpkg` and other tooling. It might be possible to get rid of it, but I would do that in some other PR :smiley:

Reply:
> setup_env_in_openshift.sh is also used in run_worker.sh and run_tests.sh

Yes, but there's a separate [install-deps-worker.yaml](https://github.com/packit/packit-service/blob/main/files/install-deps-worker.yaml#L12) for them. 

> it's required by fedpkg and other tooling

I don't see anything in `dnf repoquery --whatrequires nss_wrapper` output on my machine.

Reply:
Removed. Thanks!